### PR TITLE
Set up api, light and dark mode css variables

### PR DESCRIPTION
**Pull request by Vutlhari-Mashimbyi**

### CODE REVIEW

**Review by sudosf — CHANGES_REQUESTED** [review]
remove variables.css

**Review by tebogo-bbd — DISMISSED** [review]
please remove everything on backend 😅

File: frontend/css/style.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    --text-correct: #177915;
+    --primary:#9E092A;
+    --secondary: #434A5A;
+    --accent : #A5CAD7;
+    --card-bg: #202528;
+    --select : #7DD0F3;
+  }
+
+
+@media (prefers-color-scheme: dark) {
```
Comment (sudosf) [inline frontend/css/style.css:24]: will review later, plz comment/remove for now

File: frontend/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <script src="js/index.js"></script>
```
Comment (sudosf) [inline frontend/index.html]: to ensure the script is loaded after the page, please update to:

    <script defer src="js/index.js"></script>

**Review by sudosf — CHANGES_REQUESTED** [review]

File: frontend/css/style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    --primary-color:#9E092A;
+    --secondary-color: #434A5A;
+    --accent-color : #A5CAD7;
+    --card-bg-color: #202528;
+    --select-color : #7DD0F3;
+    }
+  }
+*/
+body {
+    color: var(--text);
```
Comment (sudosf) [inline frontend/css/style.css]: please update variable to match the new one :)

**Review by sudosf — CHANGES_REQUESTED** [review]

File: frontend/css/style.css (hunk 3 above)
Comment (Vutlhari-Mashimbyi) [reply frontend/css/style.css]: Did not see that one 😃

File: frontend/css/style.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
+:root {
+    --text-color: #0A0707;
+    --bg-color: #FDFCFC;
+    --text-correct-color: #177915;
+    --primary-color:#903634;
```
Comment (sudosf) [inline frontend/css/style.css]: plz add small space b4 the hex ..color: #9....

File: frontend/css/style.css [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    --accent-color : #7DD0F3;
+    --card-bg-color: #E2E8F0;
+    --select-color : #7DD0F3;
+  }
+  
+  .dark-mode {
+    --text-color: #DBF6FD;
+    --bg-color: #181B1C;
+    --text-correct-color: #177915;
+    --primary-color:#9E092A;
```
Comment (sudosf) [inline frontend/css/style.css]: plz space b4 the hex code

**Review by sudosf — APPROVED** [review]

**Pull request merged**